src/rgw/rgw_lua.cc

```diff
@@ -5,11 +5,6 @@
 #include "rgw_lua_utils.h"
 #include "rgw_sal_rados.h"
 #include "rgw_lua.h"
-#ifdef WITH_RADOSGW_LUA_PACKAGES
```

Comment:
why was this removed?
this is used to support systems that do not have luarocks (IIRC, opensuse, freebsd)
also, this could be used by downstream packagers that don't want to allow for any external lua libs (for security reasons)